src/coreComponents/mesh/generators/CellBlockManager.cpp

```diff
@@ -53,196 +54,111 @@ ArrayOfArrays< localIndex > CellBlockManager::getNodeToElements() const
 
   // First step: how many elements for each node, stored in the elemsPerNode array.
   array1d< localIndex > elemsPerNode( m_numNodes );
-  RAJA::ReduceSum< parallelHostReduce, localIndex > totalNodeElems = 0;
+  elemsPerNode.setValues< serialPolicy >( elemMapExtraSpacePerNode() );
 
   for( localIndex iCellBlock = 0; iCellBlock < numCellBlocks(); ++iCellBlock )
   {
-    const CellBlockABC & cb = this->getCellBlock( iCellBlock );
-    array2d< localIndex, cells::NODE_MAP_PERMUTATION > const & elemToNode = cb.getElemToNodes();
-    forAll< parallelHostPolicy >( cb.numElements(), [&elemsPerNode, totalNodeElems, &elemToNode, &cb] ( localIndex const k )
+    CellBlock const & cb = this->getCellBlocks().getGroup< CellBlock >( iCellBlock );
+    arrayView2d< localIndex const, cells::NODE_MAP_USD > const elemToNode = cb.getElemToNode();
+    forAll< parallelHostPolicy >( cb.numElements(), [elemsPerNode = elemsPerNode.toView(),
+                                                     elemToNode, &cb] ( localIndex const cellID )
```

Comment:
I would think that 'cellIndex' would be better than `cellID`. I don't know...what is your preference in this case?

Reply:
Yes, I think I agree. We had a mix and match of naming styles across this part of the code, with `k`, `iCell`, `cellID`, `cellIndex`, etc. I tried to consistify and pick what seemed most commonly used, but it wasn't my top preference either. I'll go ahead and replace across all of `mesh` component.